read-fonts/src/tables/svg.rs

```diff
@@ -0,0 +1,135 @@
+//! The [SVG](https://learn.microsoft.com/en-us/typography/opentype/spec/svg) table
+
+use core::cmp::Ordering;
+
+include!("../../generated/generated_svg.rs");
+
+/// An [SVG document](https://learn.microsoft.com/en-us/typography/opentype/spec/svg). Is not
+/// guaranteed to be valid and might be compressed.
+pub struct SVGDocument<'a>(&'a [u8]);
```

Comment:
This type seems unnecessary to me since we don’t expect to add any additional functionality here. I suggest just returning the slice directly from `glyph_data` and adding a comment there describing the content.